projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -910,6 +910,31 @@ public function render_date_field( $id, $label, $value, $class, $required, $requ
 		return $field;
 	}
 
+	/**
+	 * Return the HTML for the file field.
+	 *
+	 * @param string $id - the ID.
+	 * @param string $label - the label.
+	 * @param string $class - the field class.
+	 * @param bool   $required - if the field is marked as required.
+	 * @param string $required_field_text - the text in the required text field.
+	 *
+	 * @return string HTML
+	 */
+	private function render_file_field( $id, $label, $class, $required, $required_field_text ) {
+		$field  = $this->render_label( 'file', $id, $label, $required, $required_field_text );
+		$field .= "<input 
+			type='file' 
+			name='" . esc_attr( $id ) . "' 
+			id='" . esc_attr( $id ) . "' 
+			" . $class . '
+			' . ( $required ? "required aria-required='true'" : '' ) . "
+			accept='.pdf,.jpg'
+			style='" . $this->field_styles . "'
+		/>\n";
```

Comment:
We now have logic in the process_file_uploads() method below to handle multiple files. We could add that here simply enough by including a 'multiple' prop. 

I did try that locally. Doing so allows me to upload multiple files. And the form submission is successful. But we still only show one file in the form data on the Feedback page.